pod-operation/src/demo.rs

```diff
@@ -14,6 +17,8 @@ pub async fn blink(mut signal_light: SignalLight) {
 			signal_light.disable();
 		}
 
+		println!("Current reading: {:?}", pt.read());
+
```

Comment:
Chore: could we create a separate function for demonstrating the pressure transducer? My thought with these demo functions was to provide independent demonstrations of single components: if we wanted to test the signal light on its own, we wouldn't want to require also having an I2C device wired up. We might also need to test different components at different sampling rates: we wouldn't want to strobe the signal light just to measure another sensor.